### DISCUSSION TITLE
thread/wangzhe-thread

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#18] [Announcements]
# thread/wangzhe-thread

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/thread/wangzhe-thread.html

**Comment by zqs198739** [comment]
二哥，3种线程创建方式，你只写了3种？Callable呢？
